charts/kong/values.yaml

```diff
@@ -121,10 +121,10 @@ extraLabels: {}
 # Specify Kong's Docker image and repository details here
 image:
   repository: kong
-  tag: "3.0"
+  tag: "3.1"
```

Comment:
Are we sure we want to bump the gateway straight away given we have no compatibility tests passed yet? https://github.com/Kong/kubernetes-ingress-controller/issues/3220

Reply:
Wait, we didn't do that before release?

Reply:
Well, whatever, let https://github.com/Kong/kubernetes-ingress-controller/pull/3288#issuecomment-1358290461 run and if it passes we're good to go here.